src/modules/map/components/attributionInfo/AttributionInfo.js

```diff
@@ -25,32 +25,47 @@ export class AttributionInfo extends BaElement {
 		const translate = (key) => this._translationService.translate(key);
 		const { active, zoom } = this._state;
 
-		const geoResource = active[0] ?  this._georesourceService.byId(active[0].id) : null;
+		const attributionsRaw = new Array(); 
 
-		if (!geoResource) {
-			return nothing;
-		} 
+		for (const layer of active)  {
+			if (!layer.visible) {
+				continue;
+			} 
+			
+			const geoResource = this._georesourceService.byId(layer.id);
+
+			if (!geoResource) {
+				continue;
+			} 
 
-		const attributions = geoResource.getAttribution(zoom);
+			attributionsRaw.push(geoResource.getAttribution(zoom));
+		}
 
-		if (!attributions) {
+
+		// render fallback if no layers are set or visible
+		if (attributionsRaw[0] === null || attributionsRaw[0] === undefined) {
 			return html`
             		<div>${translate('map_attributionInfo_fallback')}</div>
 				`;
 		}
```

Comment:
I think we can remove the complete fallback case now, it's not needed anymore...